features/polyExplorer/src/components/clusterStory/clusterStory.css

```diff
@@ -0,0 +1,67 @@
+.cluster-story .content {
+    width: 100%;
+    padding: 0 16px;
+    box-sizing: border-box;
+    z-index: 2;
+}
+
+.cluster-story .content * {
+    max-width: var(--max-width);
+    color: var(--color-text-dark);
+}
+
+.cluster-story .content p {
+    line-height: 120%;
+    letter-spacing: -0.01em;
+}
+
+.cluster-story .scroll-up-btn,
+.cluster-story .progress-bar {
+    position: absolute;
+    top: 0;
+    left: 0;
+}
+
+.cluster-story .progress-bar {
+    height: 4px;
+}
+
+.cluster-story .scroll-up-btn {
+    width: 100%;
+    height: 10%;
+}
+
+.cluster-story .content::-webkit-scrollbar-track {
+    margin-top: 8px;
+}
+
+.cluster-story-main-title {
+    font-size: 24px;
+    font-weight: 600;
```

Comment:
Is this taking into account @lenalaupoly Big Font Unification?

Reply:
Not yet, she's not addressing this part of the polyExplorer until we merge this branch in `main`.